src/utils/notEmpty.ts

```diff
@@ -0,0 +1,3 @@
+export default function notEmpty<T>(value: T | null | undefined): value is T {
+  return value !== null && value !== undefined;
+}
```

Comment:
Where are you using this? Copypaste error?

Reply:
It was meant to be used during the normalization of the Lessons query but I decided that it would fit better in the same file (models/Lesson) since I did not use the function anywhere else. It has been solved in the new commit where I delete this file.